index.js

```diff
@@ -4,9 +4,64 @@
  * @description Module `jupyter-paths` provides path helpers for IPython 4.x
  */
 
+const fs = require('fs');
 const path = require('path');
 const home = require('home-dir');
-const sysPrefixPromise = require('sys-prefix-promise')
+const sysPrefixPromise = require('sys-prefix-promise');
```

Comment:
Thanks for cleaning up my loosey-goosey semi colons while you were at it. :wink: